geotrek/api/v2/serializers.py

```diff
@@ -274,6 +274,8 @@ def get_attachment_file(self, obj):
 
     def get_thumbnail(self, obj):
         thumbnailer = get_thumbnailer(self.get_attachment_file(obj))
+        if hasattr(obj, 'is_image') and not obj.is_image:
```

Comment:
I think that attachment instance has always is_image attribute

Reply:
`AccessibilityAttachment` also uses this serializer, and does not have the attribute. It raised an error in one of the tests